lib/cn/csched_sp3_work.c

```diff
@@ -419,6 +422,11 @@ sp3_work_leaf_len(
         *rule = CN_CR_LLONG;
 
         list_for_each_entry_reverse (le, head, le_link) {
+            uint vgroups = kvset_get_vgroups(le->le_kvset);
```

Comment:
const :)

Reply:
eliminated due to your following suggestion